Core/include/Acts/Seeding/BinnedSPGroup.hpp

```diff
@@ -273,11 +273,11 @@ class BinnedSPGroup {
   template <typename spacepoint_iterator_t>
   BinnedSPGroup<external_spacepoint_t>(
       spacepoint_iterator_t spBegin, spacepoint_iterator_t spEnd,
-      GlobalPositionFunctor toGlobal,
+      GlobalPositionFunctor& toGlobal,
       std::shared_ptr<Acts::BinFinder<external_spacepoint_t>> botBinFinder,
       std::shared_ptr<Acts::BinFinder<external_spacepoint_t>> tBinFinder,
       std::unique_ptr<SpacePointGrid<external_spacepoint_t>> grid,
-      Acts::Extent rRangeSPExtent,
+      Acts::Extent& rRangeSPExtent,
```

Comment:
will that be changed inside, otherwise it should be `const` right?

Reply:
It gets modifies:
```cpp
    // store x,y,z values in extent
    rRangeSPExtent.extend({spX, spY, spZ});
```